server/sonar-web/src/main/js/apps/marketplace/components/PluginDescription.tsx

```diff
@@ -27,24 +27,14 @@ interface Props {
 }
 
 export default class PluginDescription extends React.PureComponent<Props> {
-  handleCategoryClick = (e: React.SyntheticEvent<HTMLAnchorElement>) => {
-    e.preventDefault();
-    this.props.updateQuery({ search: this.props.plugin.category });
```

Comment:
`updateQuery` can now be removed from the props. Also check the parent components, they might not need this prop as well.

Reply:
and actually this component should become functional